### ISSUE TITLE
Allmanga

### ISSUE THREAD

**Issue by SoloJovian** [tachiyomiorg/tachiyomi-extensions#17848]
### Source name

Allmanga

### Source link

allmanga.to

### Source language

Engish

### Other details

18+/NSFW = yes

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

**Comment by AwkwardPeak7** [comment]
This is just a mirror of AllAnime, and uses same api. So no reason to add it even as a mirror domain.

**Comment by SoloJovian** [comment]
> This is just a mirror of AllAnime, and uses same api. So no reason to add it even as a mirror domain.

Yeah, just saw but man [All Anime](https://allanime.to/) is the domain for only anime, the domain for manga is [All Manga](https://allmanga.to/). It's confusing if you have the name of a Anime website in the extension list of an app where you can only read manga. If possible please change the name of the extension to AllManga.
